configtx/example_test.go

```diff
@@ -352,6 +352,33 @@ func Example_organizations() {
 	o.RemoveOrganization("OrdererOrg2")
 }
 
+// This example shows updating the individual orderer configuration values.
+func ExampleOrdererGroup_SetConfiguration_individual() {
+	baseConfig := fetchChannelConfig()
+	c := configtx.New(baseConfig)
+	o := c.Orderer()
+
+	err := o.BatchSize().SetMaxMessageCount(500)
+	if err != nil {
+		panic(err)
+	}
+
+	err = o.AddConsenter(orderer.Consenter{Address: orderer.EtcdAddress{
+		Host: "host1",
+		Port: 7050},
+		ClientTLSCert: generateCert(),
+		ServerTLSCert: generateCert(),
+	})
+	if err != nil {
+		panic(err)
+	}
+
+	err = o.EtcdRaftOptions().SetElectionInterval(50)
```

Comment:
Hmmm, what exactly does an election interval of 50 mean? Seems odd that fabric-config and/or fabric doesn't use a duration.

Reply:
Election tick is not a duration it's the number of Node.Tick invocations that must pass between elections per [docs](https://hyperledger-fabric.readthedocs.io/en/latest/raft_configuration.html?highlight=electiontick#channel-configuration).